### PR TITLE
Python script to cleanup argo and services (cloud endpoints)

### PR DESCRIPTION
**Pull request by jlewi**
Add a python function to GC old Argo workflows and GCP services (endpoints).
    
 Related to
    
 kubeflow/testing#53 GC old Argo Workflows
 kubeflow/testing#87 cron job to GC old resources.
#268 Maximum number of services reached in kubeflow-ci

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/261)

### CODE REVIEW

**Comment by jlewi** [comment]
Need to submit #251 and then rebase this PR

**Comment by jlewi** [comment]
This is ready for review.
/assign @kunmingg 
/assign @richardsliu 
/hold cancel

File: py/kubeflow/testing/cleanup_ci.py [hunk 1]
```diff
@@ -22,6 +25,95 @@ def is_match(name):
...
+          is_expired = True
+
+      if is_expired:
+        logging.info("Deleting service: %s", name)
+        is_expired = True
+        services.delete(serviceName=name).execute()
+        expired.append(name)
+      else:
+        unexpired.append(name)
+    if not "nextPageToken" in results:
```
Comment (richardsliu) [inline py/kubeflow/testing/cleanup_ci.py]: If there is no nextPageToken then don't you want to break out of the loop?

Comment (jlewi) [reply py/kubeflow/testing/cleanup_ci.py]: Good catch. PTAL.

**Pull request merged**